### ISSUE TITLE
F0038: performance regression

### ISSUE THREAD

**Issue by rvjansen** [adesutherland/CREXX#301]
develop is faster than f0038 at the moment. need to find out what causes the slowdown.

**Comment by adesutherland** [comment]
This is weird! Hopefully a diff in the interpreter will give us a clue. A couple of things to verify ...

- is the rxbin produced the same (just to make sure it is not a compiler/assembler issue)
- is the cmake configuration the same for the interpreter - there are some extra optimisations I put in (I think) 
- everything is "release" I know you will have triple checked this :-)

**Comment by rvjansen** [comment]
I am planning to profile it later today, my suspicion at the moment is changes in the date() function, but everything I say is premature at the moment. Cmake conf should be the same as this was merged into f0038. And yes, release everywhere.

**Comment by rvjansen** [comment]
Progress: less than I wanted. I need Linux for good profiling, it seems that Apple has rendered DTRACE quite useless without time consuming fidgeting with security settings. Installed QEMU and UTM on the Mac Mini M1 and to my great astonishment the timings are slightly faster than native macOS on the same machine. Only today will I have the opportunity to benchmark, but as of now I have only more question marks. The new linkage is great!

**Comment by rvjansen** [comment]
Progress: after staring long and hard at the traces, I found the sources to be different; a bug in the development version was fixed in feature f0038. Merge was done correctly and did not regress the fix. The performance difference is now <0.03 s, hardly worth the effort. Because I do not have all the counters I want in virtualized aarch64 Ubuntu, I will repeat the tests somewhere this weekend, but for the moment, nothing to see.
